test/test-ExperimentProvider.js

```diff
@@ -1,14 +1,267 @@
 "use strict";
```

Comment:
Mainly it's because the `data` object is now directly accessing prefs to look up values for experiments. So I needed to write a mock of prefs in order to keep the content-side tests. This just seemed to defeat the purpose of the tests to me, but I'm happy to put them back in content if there's a strong reason to.